examples/local/xgboost_local.py

```diff
@@ -16,7 +16,7 @@
 
 def run_experiment(on_travis: bool = False):
     task_ids = get_openmlcc18_taskids()
-    for task_no, task_id in enumerate(task_ids):
+    for task_no, task_id in enumerate(task_ids[:1]):
```

Comment:
this would only use the first task_id. Is this an artifact from testing?

Reply:
Thanks, yes, sorry.